native/cocos/audio/android/tinysndfile.h

```diff
@@ -21,8 +21,13 @@
 // much smaller and has an Apache 2.0 license.
 // The API should be familiar to clients of similar libraries, but there is
 // no guarantee that it will stay exactly source-code compatible with other libraries.
-
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 #include <sys/cdefs.h>
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+#include <cstdint>
+#endif
+#include "audio/android/utils/Compat.h"
```

Comment:
Why do we need Compat for Android?

Reply:
Openharmony's __BEGIN_DECLS definition is here